maintenance/altering.rst

```diff
@@ -0,0 +1,25 @@
+.. _upgrade:
+
+Altering for critical errors
+============================
+
+Ensuring uninterrupted operation of the Firely Server in production is crucial, as business processes built on top of the FHIR server require high availability.
+
+To quickly identify and respond to the causes of unplanned downtime, we recommend integrating Firely Server with a monitoring system capable of sending real-time notifications for critical errors.
+
+We suggest using `Seq <https://docs.datalust.co/docs/alerts>`_ and its alerting capabilities for this purpose. To reduce the risk of false positives, it is advisable to exclude all requests that return an HTTP status code 501 – Not Implemented from triggering alerts.
```

Comment:
I have two thoughts. 
1) Producing opentelemetry traces and metrics opens up more options that just log analysis. Should this be mentioned?
2) Related to that  I am not sure about the "suggest" verb. Seq is one option. I am pretty confident that you can also set this up with Azure Application Insight (either with logs, traces or metrics) or the equivalent for other cloud providers.

Reply:
1. I think until we have further recommendations on concrete queries that should be used as the source for alerts here, I'd leave it out for now.

Reply:
2. Changed the wording to highlight the advantages of Seq. I'd leave out Azure Application Insight for now until we investigate it as well.